src/DataSets/ChartData.ts

```diff
@@ -0,0 +1,13 @@
+import { wfa_g_0_5_z } from "./WhoStandardDataSets"
+
+export const chartData = {
+  "Weight-for-age GIRLS": {
+    datasets: {
+      "Girls0to5Years": wfa_g_0_5_z,
+
+    },
+    metadata: {
+      "Girls0to5Years": { label: "Girls 0 to 5 years", yaxis: "Weight (kg)", unit: "Months", range: { start: 24, end: 60 } },
```

Comment:
Girls 0 to 5 years should have a range starting at 0 and ending with 60
```suggestion
      "Girls0to5Years": { label: "Girls 0 to 5 years", yaxis: "Weight (kg)", unit: "Months", range: { start: 0, end: 60 } },
```